web-ui/view/documentation/step_page.ml

```diff
@@ -3,5 +3,94 @@ open Tyxml.Html
 let show =
   [
     div ~a:[ a_id "step-page" ] [ h2 [ txt "Step page walkthrough" ] ];
-    p [ txt "The step page" ];
+    p
+      [
+        txt
+          "The step page shows the results of running a step on OCaml-CI. If a \
+           step has failed, typically the step's page will help understand and \
+           diagnose the cause of the failure. The logs displayed on this page \
+           contain details of exactly what OCaml-CI did to execute the step \
+           and will contain information such as stack traces and/or error \
+           messages that will help identify the issue.";
+      ];
+    img
+      ~a:[ a_class [ "border border-solid" ] ]
+      ~src:"/images/step-page-overview.png" ~alt:"step-page-overview" ();
+    div [ h3 [ txt "Salient information regarding the step" ] ];
```

Comment:
We generally capitalise headers and titles, but we can do that for just h1 and h2 if you prefer, leaving h3’s sentence case. What do you think?

Reply:
This sounds good to me. Will adopt this style from here on.